src/ios/withNotificationsXcodeProject.ts

```diff
@@ -291,9 +291,9 @@ const updateNseEnv = (
 
   if (options.pushNotification?.env?.region) {
     let region = '';
-    if (options.pushNotification?.env?.region === 'us') {
+    if (options.pushNotification?.env?.region?.toLowerCase() === 'us') {
       region = 'Region.US';
-    } else if (options.pushNotification?.env?.region === 'eu') {
+    } else if (options.pushNotification?.env?.region?.toLowerCase() === 'eu') {
       region = 'Region.EU';
     }
```

Comment:
Can we maybe avoid the changing? how does this look? 

```suggestion
     const regionMap = {
  us: 'Region.US',
  eu: 'Region.EU',
};

const region = options.pushNotification?.env?.region?.toLowerCase();
const mappedRegion = regionMap[region] || '';
```